lua/efmls-configs/formatters/protolint.lua

```diff
@@ -0,0 +1,17 @@
+-- Metadata
+-- languages: proto
+-- url: https://github.com/yoheimuta/protolint
+
+local fs = require('efmls-configs.fs')
+
+local formatter = 'protolint'
+local command = string.format("%s lint -fix '${INPUT}'", fs.executable(formatter))
+
+return {
+  formatCommand = command,
+  formatStdin = true,
```

Comment:
Since it only takes the filename from the command line (`${INPUT}`) we can set this to `false`

Reply:
Updated! Thank you for the suggestion.